### ISSUE TITLE
Bugs fixed in the naval warfare branch should be marked as such

### ISSUE THREAD

**Issue by tothxa** [widelands/widelands#6141]
**Is your feature request related to a problem? Please describe.**
It's hard to keep track of open bugs because bugs that are already fixed in the naval warfare branch are not closed. Unfortunately this makes sense for bugs that also affect master until naval warfare is merged.

Moving bugs to "Done" in the "Naval Warfare" project page is not visible in the main bug list.

**Describe the solution you'd like**
Could we add a new bug tag "fixed in naval warfare" with a well visible, e.g. bright green background?
Bugs that were introduced in the naval warfare branch and are already fixed should be closed in addition, as they don't affect master anyway.

**Implementation steps**
- [x] Define tag
- [x] Tag fixed bugs
- [ ] Close bugs introduced and fixed in naval warfare

**Comment by Noordfrees** [comment]
[Fixed.](https://github.com/widelands/widelands/labels/fixed)

**Comment by tothxa** [comment]
Thank you! :)

How about the last one? Should I do it or leave them as they are and just close this bug?

**Comment by Noordfrees** [comment]
Let's just leave them open, imho.

**Comment by bunnybot** [comment]
<img align='left' src='https://codeberg.org/avatars/a4ad2f9a2504f052397d8ba82ed48ba1' alt='Nordfriese' height='48' />*[Mirrored from Codeberg](https://codeberg.org/wl/widelands/issues/4464#issuecomment-1240231)*
*On Sun Sep 24 09:35:52 CEST 2023, **Benedikt Straub (Nordfriese)** wrote:*

---

The naval warfare project tracker on GitHub has been replaced by the one on Codeberg: https://codeberg.org/wl/widelands/projects/5371

Projects are so different on GH and CB that they are not mirrored by the bot.
